src/types/mod.rs

```diff
@@ -502,6 +502,115 @@ mod rewards_overview {
     }
 }
 
+pub mod block_certificates {
+    use concordium_base::hashes::BlockHash;
+
+    use super::*;
+
+    /// An aggregate signature on a `QuorumCertificate` created
+    /// by members of the finalization committee.
+    #[derive(Debug)]
+    pub struct QuorumSignature(pub [u8; 48]);
+
+    /// A quorum certificate on a block.
+    #[derive(Debug)]
+    pub struct QuorumCertificate {
+        /// `Hash` of the block that this `QuorumCertificate`
+        /// certifies.
+        pub block_hash:          BlockHash,
+        /// `Round` of the block that this `QuorumCertificate`
+        /// certifies.
+        pub round:               Round,
+        /// `Epoch` of the block that this `QuorumCertificate`
+        /// certifies.
+        pub epoch:               Epoch,
+        /// The aggregate signature on the block identified by the
+        /// `block_hash` which serves as a proof that the block
+        /// was accepted on the chain by a quorum of the finalization
+        /// committee.
+        pub aggregate_signature: QuorumSignature,
+        /// The baker ids of the finalizers that formed
+        /// the `aggregate_signature`.
+        pub signatories:         Vec<BakerId>,
```

Comment:
I can see that this is also a list in Haskell, but semantically this is a set, so can you elaborate on that.